### ISSUE TITLE
Webpack 2 and -minimize

### ISSUE THREAD

**Issue by jrgleason** [webpack-contrib/html-loader#112]
Currently I have the following implementation...

    {
          test: /\.html$/,
          loaders: ["html?interpolate=require&-minimize"]
    },

In Webpack 2 you are supposed to something like this...

    {
          test: /\.html$/,
          use: [{loader:"html-loader",options:{interpolate:"require"}}]
    },

However this pattern does not seem to support the `-minimize` property

**Comment by jrgleason** [comment]
Even stranger, if I try to use it the right way without the -minimize (AKA the webpack 2 way). It doesn't seem to parse some angular stuff right but if I replace it with the old way like this...

```
{
          test: /\.html$/,
          use: ["html-loader?interpolate=require&-minimize"]
},
```

It works :-/

**Comment by hemanth** [comment]
Thanks for reporting, this is fixed in #116.
